src/page/gjennomforing-detaljer/endringsmeldinger/endringsmelding/EndringsmeldingPanel.tsx

```diff
@@ -11,16 +11,17 @@ import classNames from 'classnames'
 import { Endringsmelding, EndringsmeldingStatus, EndringsmeldingType } from '../../../../api/schema/endringsmelding'
 import { EndringsmeldingIkon } from './EndringsmeldingIkon'
 import { EndringsmeldingInnhold } from './innhold/EndringsmeldingInnhold'
+import { VarighetValg } from './VarighetSelect'
 
 
 interface EndringsmeldingProps {
 	endringsmelding: Endringsmelding
 	onFerdig: () => void
-	varighet: number | null
+	valgtVarighet: VarighetValg
 	className?: string
 }
 
-export const EndringsmeldingPanel = ({ endringsmelding, onFerdig, varighet, className }: EndringsmeldingProps): React.ReactElement => {
+export const EndringsmeldingPanel = ({ endringsmelding, onFerdig, valgtVarighet: varighet, className }: EndringsmeldingProps): React.ReactElement => {
```

Comment:
Her kan man godt bare kalle valgtVarighet for varighet, om man uansett ønsker å bruke navnet videre

Reply:
Jeg blir ikke helt enig med meg selv om navnene, det er noe som skurrer litt for meg. Men tror du det er bedre nå?